config/spryk/spryks/Spryker/Zed/Business/AddCrudFacade.yml

```diff
@@ -22,3 +22,4 @@ preSpryks:
     - AddModule
 
 postSpryks:
+    - AddZedPersistencePropelSchemaProperty
```

Comment:
You also need to add the arguments. This Spryk should create an XML file with the defined entities in https://spryker.atlassian.net/wiki/spaces/AOP/pages/3224371207/WIP+APPS-3483+CRUD+Facade+function+RFC+to+Spryk#Domain-model-diagram

Reply:
Do we need to leak the Propel part here everywhere?
I find it clear enough as persistence schema to be honest

> AddZedPersistenceSchemaProperty